kong/db/schema/typedefs.lua

```diff
@@ -449,10 +452,12 @@ local function validate_path_with_regexes(path)
   -- We can't take an ok from validate_path as a success just yet,
   -- because the router is currently more strict than RFC 3986 for
   -- non-regex paths:
```

Comment:
Is this comment relevant anymore or perhaps wrong?

Reply:
You are right. It doesn't hold now.